src/style.css

```diff
@@ -1,3 +1,52 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  padding-top: 10px;
+}
+
+.main_nav-link {
+  display: flex;
+  gap: 20px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
+  margin-right: 50px;
+  height: 60px;
+}
```

Comment:
Be consistent with your vertical margins. You have a `margin-right` on `.main_nav-link` but no vertical margins are defined. If you want to add space between elements vertically, choose either top or bottom margin but not both to avoid margin collapsing issues.